### DISCUSSION TITLE
视频列表下载报错

### DISCUSSION THREAD

**Discussion by Refarin** [yutto-dev/yutto#75] [Q&A] [answered]
视频列表 无论是给出的例子还是找的https://www.bilibili.com/medialist/play/479592209?business=space_collection&business_id=100808
  都报错。想问下怎么解决？
C:\Users\Administrator>yutto https://www.bilibili.com/medialist/play/100969474?business=space_series&business_id=194743  -b
 INFO  未提供 SESSDATA，无法下载会员专享剧集哟～
 ERROR  url 不正确，也许该 url 仅支持批量下载，如果是这样，请使用参数 -b～
 INFO  已终止下载，再次运行即可继续下载～
'business_id' 不是内部或外部命令，也不是可运行的程序
或批处理文件。

**Comment by SigureMo** [comment, marked as answer]
用引号包裹 url

```bash
yutto -b 'https://www.bilibili.com/medialist/play/479592209?business=space_collection&business_id=100808'
```

**Reply by Refarin** [reply]
感谢
